query-node/mappings/content-directory/transaction.ts

```diff
@@ -88,19 +89,10 @@ export async function contentDirectory_TransactionFailed(db: DB, event: Substrat
   const failedOperationIndex = event.params[1].value as number
   const operations = decode.getOperations(event)
 
-  if (operations.length === 0 || operations.length === 1) return
+  const successfulOperations = operations.toArray().slice(2, failedOperationIndex)
```

Comment:
why are we always skipping first two operations?

Reply:
If `operations` length is `0` then there is no successful operation, if the length is `1` then there is only one operation which is the failed one (that's why the `TransactionFailed` event is triggered for). In both cases, there is nothing for me to process.

Reply:
Okay, that's wrong! I am updating now! I mixed the indices with the length